packages/web/components/DataTable/index.tsx

```diff
@@ -177,17 +178,22 @@ export default function DataTable({
 
   useEffect(() => {
     if (storageKey) {
-      if (localStorage.getItem(`${storageKey}-page-size`)) {
-        const storagePageSize = parseInt(localStorage.getItem(`${storageKey}-page-size`) || '', 0);
-        setPageSize(storagePageSize);
+      const storagePageSize = localStorage.getItem(`${storageKey}-page-size`);
+      if (storagePageSize) {
+        const parsedPageSize = parseInt(localStorage.getItem(`${storageKey}-page-size`) || '', 0);
```

Comment:
```suggestion
        const parsedPageSize = parseInt(storagePageSize || '', 0);
```

Apply to the others as well